=== modified file 'user_docs/de/userGuide.t2t'
--- a/user_docs/de/userGuide.t2t	2013-01-15 06:57:31 +0000
+++ b/user_docs/de/userGuide.t2t	2013-01-20 20:04:26 +0000
@@ -1684,8 +1684,8 @@
 | Statuszeile ausgeben | l2+ab |
 | zum übergeordneten Objekt springen | auf2 |
 | zum ersten untergeordneten Objekt springen | ab2 |
-| zum nächsten objekt bewegen | links2 |
-| zum vorigen Objekt bewegen | rechts2 |
+| zum nächsten objekt bewegen | rechts2 |
+| zum vorigen Objekt bewegen | links2 |
 | Textformatierungen angeben | obere Routingreihe |
 
 

